fake-ipa/fake_ipa/inspector.py

```diff
@@ -30,12 +30,13 @@ def inspect(system, inspection_callback_url, verify, cert, logger):
                     "lldp": None,
                     "product": "0x0001",
                     "vendor": "0x1af4",
-                    "name": "eth1",
+                    "name": nic.get("name", f"enp1s{i}"),
                     "has_carrier": True,
-                    "ipv4_address": system.get("nics")[0]["ip"] or None,
+                    "ipv4_address": nic.get("ip"),
                     "client_id": None,
-                    "mac_address": system.get("nics")[0]["mac"],
+                    "mac_address": nic["mac"],
                 }
+                for i, nic in enumerate(system.get("nics"))
                 ],
```

Comment:
```suggestion
            ],
```
Also L46 and L47 have 4 too much indents.

Reply:
fixed